example/tutorials/tutorial.md

```diff
@@ -112,6 +112,25 @@ Examine the logs to see that it is properly running.
 docker stop kubearmor-otel-receiver; docker rm kubearmor-otel-receiver
 ```
 
+### Kubearmor receiver config.
```

Comment:
Let's also refer this in the step where we create the deployments -
https://github.com/Chinwendu20/OTel-receiver/blob/e8b8f5b53b8fd3aec9435b850d1c0a0433549d08/example/tutorials/tutorial.md?plain=1#L34
https://github.com/Chinwendu20/OTel-receiver/blob/e8b8f5b53b8fd3aec9435b850d1c0a0433549d08/example/tutorials/tutorial.md?plain=1#L75

Reply:
Also, the preview doesn't look as intended :sweat_smile: - https://github.com/Chinwendu20/OTel-receiver/blob/env/example/tutorials/tutorial.md#kubearmor-receiver-config